client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
-	sections.push( {
-		name: 'devdocs',
-		paths: [ '/devdocs' ],
-		module: 'devdocs',
-		secondary: true,
-		enableLoggedOut: true
-	} );
+sections.push( {
```

Comment:
I agree that we need to remove those feature flag checks from files where sections definition are located. At the same time I think we should move existing logic to entry point of each individual section. This means I'm suggesting to bring back those file guard to the first file which isn't generated by Webpack, to make sure we don't change any existing logic. We can remove some flags in separate PRs.